docs/platforms/python/integrations/langchain/index.mdx

```diff
@@ -36,6 +36,8 @@ An additional dependency, `tiktoken`, is required to be installed if you want to
 
 In addition to capturing errors, you can monitor interactions between multiple services or applications by [enabling tracing](/concepts/key-terms/tracing/). You can also collect and analyze performance profiles from real users with [profiling](/product/explore/profiling/).
 
+Tracing is required to see AI pipelines in sentry's [AI tab](https://sentry.io/insights/ai/llm-monitoring/).
```

Comment:
Thanks for adding this, definitely an oversight that we forgot to add this before!